src/fileattributefetch.cpp

```diff
@@ -168,7 +170,11 @@ void FileAttributeFetchChannel::failed(MegaClient* client)
         {
             // retry
             it->second->retries++;
-            it++;
+
+            // move from pending to fresh
```

Comment:
Without this code, the request isn't sent again. If the state `REQ_FAILED` persists, an error is detected in the next event loop, without retrying.
Also, the `FileAttributeFetch` object is put again in the map in which it starts.